src/components/app-configure/general/DeleteTimerPopup.vue

```diff
@@ -0,0 +1,119 @@
+<script lang="ts" setup>
+import { ref } from 'vue'
+import Vue3CircleProgress from 'vue3-circle-progress'
+
+import VButton from '@/components/lib/VButton/VButton.vue'
+
+const emit = defineEmits(['cancel', 'delete'])
+
+const timer = ref(59)
+let timerProgress = 59000
+const initialProgressPercentage = (timerProgress / 60000) * 100
+const timerProgressPercentage = ref(initialProgressPercentage)
+
+const timerInterval = setInterval(() => {
+  --timer.value
+  if (timer.value === 0) {
+    proceedDelete()
+  }
+}, 1000)
+
+const progressInterval = setInterval(() => {
+  timerProgress = timerProgress - 10
+  timerProgressPercentage.value = (timerProgress / 60000) * 100
+}, 10)
+
+function clearTimer() {
+  clearInterval(timerInterval)
+  clearInterval(progressInterval)
+}
+
+function handleCancelDelete() {
+  clearTimer()
+  emit('cancel')
+}
+
+function proceedDelete() {
+  clearTimer()
+  emit('delete')
+}
```

Comment:
@shrinathprabhu @karthik-durai 

Let's follow a consistent naming convention for event handlers. Few possible options:

For `event`, name handler as `handle[Event]`
```
@click="handleCancellation"
@click="handleProceed"
```

For `event`, name handler as `[event]EventHandler`:
```
@click="cancelEventHandler"
@click="proceedEventHandler"
```

Which one (or any other) should we adopt?

Reply:
I would prefer handle[Event]